File: Selenium/Activities/activity6.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains
from selenium.webdriver.common.keys import Keys

with webdriver.Chrome() as driver:
    actions = ActionChains(driver)
    driver.get("https://v1.training-support.net/selenium/input-events")

    print("Page title is: ", driver.title)
    actions = ActionChains(driver)

    actions.send_keys("R").key_down(Keys.CONTROL).send_keys("a").send_keys("c").key_up(Keys.CONTROL).perform()
    driver.close

File: Selenium/Activities/Activity6.java
package activities;

import org.openqa.selenium.Keys;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity6 {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		Actions builder = new Actions(driver);
		
		driver.get("https://v1.training-support.net/selenium/input-events");
		System.out.println(driver.getTitle());
		
		builder.sendKeys("R").build().perform();
		builder
		       .keyDown(Keys.CONTROL)
		       .sendKeys("a")
		       .sendKeys("c")
		       .keyUp(Keys.CONTROL)
		       .build()
		       .perform();
		
		
	}

}
